ui/test/nuxeo-favorites-toggle-button.test.js

```diff
@@ -48,16 +49,29 @@ suite('nuxeo-favorites-toggle-button', () => {
       ]);
     });
 
+    teardown(() => {
+      if (window.confirm.restore) {
+        window.confirm.restore();
+      }
+    });
+
     test('it should display the document as favorite', () => {
       expect(element.favorite).to.be.true;
     });
 
-    test('toggle should remove document from favorites', async () => {
+    test('toggle should remove document from favorites when clicked ok', async () => {
       // Remove document from favorites by toggling
+      sinon.stub(window, 'confirm').returns(true);
       tap(element);
       await waitChanged(element, 'favorite');
       expect(element.favorite).to.be.false;
     });
+
+    test('toggle should not remove document from favorites when clicked Cancel', async (done) => {
```

Comment:
There's no need use the done here and call it later. This is usually used in sync functions.